src/student.html

```diff
@@ -1,10 +1,90 @@
 <!DOCTYPE html>
-
 <html>
     <head>
+        <link rel="stylesheet" href="index.css">
     </head>
 
-    <body id = "body">
-        <p>Student View</p>
+    <style>
+        table, th, td {
+          border:1px solid black;
+        }
+    </style>
+    <body>
+        <div class="header">
+            <h1>Concordia University</h1>
+            <p>Marks and Grades assessment for SOEN 287 for Fall 2022</p>
+        </div>
+
+        <p><h1><b>Student View</b></h1></p>
+
+        <div class="student">
+            <p><b>Here are your grades for SOEN 287:</b></p>
+            <p><b>Assignment:</b></p>
+                <p>
+                    <table>
+                        <tr>
+                            <th><b>Task 1</b></th>
+                            <th><b>Task 2</b></th>
+                            <th><b>Total</b></th>
+                        </tr>
+                        <tr>
+                            <td>5/5</td>
+                            <td>10/15</td>
+                            <td>15/20</td>
+                        </tr>    
+                    </table>
+                </p>
+            <p><b>Midterm:</b></p>
+                <p>
+                    <table>
+                        <tr>
+                            <th><b>Multiple Choice</b></th>
+                            <th><b>Short Answers</b></th>
+                            <th><b>Long Answer</b></th>
+                            <th><b>Total</b></th>
+                        </tr>
+                        <tr>
+                            <td>5/5</td>
+                            <td>12/15</td>
+                            <td>7/10</td>
+                            <td>24/30</td>
+                        </tr>
+                    </table>
+                </p>
+            <p><b>Final Exam:</b></p> 
+                <p>
+                  <table>
+                      <tr>
+                            <th><b>Multiple Choice</b></th>
+                            <th><b>Short Answers</b></th>
+                            <th><b>Long Answer</b></th>
+                            <th><b>Total</b></th>
+                        </tr>
+                        <tr>
+                            <td>8/10</td>
+                            <td>22/30</td>
+                            <td>9/10</td>
+                            <td>39/50</td>
+                        </tr>
+                    </table>
+                </p>
+            <p><b>Final Grade:</b></p> 
+                <p>
+                    <table>
+                        <tr>
+                            <th>Total</th>
+                        </tr>
+                        <tr>
+                            <td>78/100</td>
+                        </tr>
+                    </table>
+                </p>
+        </div>
+
+        <div class="footer">
+            Website made by ... names
+        </div>
     </body>
+
+    <script src = "login.js"></script>
```

Comment:
We no longer require the login.js script (it does not exist anymore) so you can remove this tag